contracts/mocks/TransferableControllerMock.sol

```diff
@@ -1,9 +1,9 @@
 pragma solidity ^0.4.19;
 
-import "./MFC_coin.sol";
-import "./ICO_controller.sol";
+import "./../MFC_coin.sol";
+import "./../ICO_controller.sol";
 
-contract ICO_controller_mock is TransferableInterface {
+contract TransferableControllerMock is TransferableInterface {
```

Comment:
We had two ICO controller mocks - ICO_controller_mock and mocks/IcoControllerMock, so I renamed this one to `TransferableControllerMock` (as it's only purpose is to have Transferable interface, we don't need all ICO controller features here).